boards/shared/Src/App/App_SharedHeartbeatMonitor.c

```diff
@@ -36,22 +38,20 @@ void App_SharedHeartbeatMonitor_Destroy(struct HeartbeatMonitor *const heartbeat
 
 bool App_SharedHeartbeatMonitor_Tick(struct HeartbeatMonitor *const heartbeat_monitor)
 {
-    static bool status = true;
-
```

Comment:
Moved to the malloced struct, since there's no way to reset static variables between tests